docs/specification/2024-11-05/basic/transports.mdx

```diff
@@ -20,7 +20,8 @@ It is also possible for clients and servers to implement
 
 In the **stdio** transport:
 
-- The client launches the MCP server as a subprocess.
+- The client launches the MCP server as a subprocess, creating a dedicated stdio channel
+  and ensuring a strict one-to-one relationship between each client and its server instance.
```

Comment:
This change should be made in `docs/specification/draft/basic/transports.mdx` instead of this file.  (I don't think it is worth changing older versions of the spec.)